contrib/review-trailers-tooling/NVIM_RUNTIME_PATH/plugin/insert-github-user-info.vim

```diff
@@ -0,0 +1,14 @@
+" Replace word under cursor (which should be a GitHub username)
+" with some user info ("Full Name <email@address>").
+" If info cout not be found, "Not found" is inserted.
+function! <SID>InsertGitHubUserInfo()
```

Comment:
Well, certainly, range handling could be added. But note that, in principle, this is not intended to edit existing trailers, which shouldn't be there if they only have the username. Rather, this is intended to enter new trailers, where you type the trailer title (or use provided UltiSnips templates to do that), and then, enter a username, and quickly transform it into fullname and email. So, I'm not sure if range handling has real sense in such a context.